pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -128,7 +133,8 @@ class LightningTensorNet:
         num_wires(int): the number of wires to initialize the device with
         c_dtype: Datatypes for tensor network representation. Must be one of
             ``np.complex64`` or ``np.complex128``. Default is ``np.complex128``
-        method(string): tensor network method. Options: ["mps"]. Default is "mps".
+        method(string): tensor network method. Supported methods are "mps" (Matrix Product State) and
+            "tn" (Exact Tensor Network). Options: ["mps", "tn"].
```

Comment:
Default "mps"?

Reply:
and can we swap the order of method and c_dtype to align with the args

Reply:
Done!